packages/community-cli-plugin/src/commands/start/attachKeyHandlers.js

```diff
@@ -46,14 +46,17 @@ export default function attachKeyHandlers({
   const onPress = async (key: string) => {
     switch (key) {
```

Comment:
Thanks! I have change the code based on your suggestions.

>Note that this will deviate us from [Expo CLI](https://github.com/expo/expo/blob/48773ffcb57f4ec4529571ff82b7546d6854961f/packages/%40expo/cli/src/start/interface/startInterface.ts#L139)

Would you suggest me to make a new PR for the same change in the Expo CLI repo?